src/components/TheTopNavbar.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-toolbar data-cy="nav-drawer" color="primary">
+    <v-toolbar-title>
+      <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
+        <v-menu  transition="slide-x-transition">
+          <template #activator="{ props }">
+            <v-btn
+              data-cy="user-menu"
+              class="hidden-xs-only d-flex text-body-1"
+              v-bind="props"
+              variant="text"
```

Comment:
Does this always show the hamburger, including on large devices? The design intended to hide the hamburger menu on large devices. I'm open to discussion on that point. What do you think is better and why?

Reply:
its the menu which open when click on username.

according to the figma file on larger devices we have to click on username to see log out option thats why it is visible for both large and small devices.